src/bootstrap/format.rs

```diff
@@ -0,0 +1,30 @@
+//! Runs rustfmt on the repository.
+
+use crate::{util, Build};
+use std::path::PathBuf;
+use std::process::Command;
+
+pub fn format(build: &Build, check: bool) {
+    let target = &build.build;
+    let rustfmt_var = match std::env::var("RUSTFMT") {
```

Comment:
We should query the environment variable in `src/bootstrap/config.rs` similar to how we setup `initial_rustc`; it'll be `Option<PathBuf>` probably.